src/app.ts

```diff
@@ -115,15 +117,17 @@ class App {
     app.post(`/did/update`, IssuerController.updateValidator, new IssuerController().updateDid)
     app.post(`/did/deactivate/:did`, IssuerController.deactivateValidator, new IssuerController().deactivateDid)
     app.get(`/did/list`, new IssuerController().getDids)
-    app.get(`/did/:did`, new IssuerController().getDid)
+    app.get(`/did/resolve/:didUrl`, new IssuerController().getDid)
 
     // Resource API
     app.post(`/resource/create/:did`, IssuerController.resourceValidator, new IssuerController().createResource)
+    app.get(`/resource/list/:did`, new IssuerController().getResourceList)
+    app.get(`/resource/dereferencing/:did/:resourceId`, new IssuerController().getResource)
```

Comment:
@abdulla-ashurov can we remove /dereferencing from this api path. It's make it complicated to understand

Reply:
We cannot rename from `resource/dereferencing/:did/:resourceId` to `resource/:did/:resourceId` because if we remove the `dereferencing` from endpoint, we end up with 2 endpoints that will be blocked for each other:
1. `/resource/:did/:resourceId`: it handles everything which comes after `resource/`
2. `/resource/list/:did` - this endpoint never works because all requests from this endpoint will be handled by the first endpoint.